byron/ledger/executable-spec/test/Main.hs

```diff
@@ -55,4 +58,5 @@ main = defaultMain tests
       , testProperty "Relevant cases are classified" UPDATE.upiregRelevantTracesAreCovered
       , testProperty "Only valid signals are generated" UPDATE.onlyValidSignalsAreGenerated
       ]
+    , testRelation
```

Comment:
These are not properties of the transition systems of the ledger, but we can leave this here for now and move it once we move the `Relation` class and instances somewhere else.